submissions/Nutan/html-forms/index.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
```

Comment:
this file name `index.html` doesn't tell what this file is for. Kindly name it to something like `contact.html`